.github/workflows/ci.yml

```diff
@@ -2,24 +2,36 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: 
+      - master
   pull_request:
-    branches: [ master ]
+    branches: 
+      - master
 
 jobs:
-  build:
-
+  Lint:
     runs-on: ubuntu-latest
 
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v1.4.4
+      with:
+        node-version: 14.x
+    - run: npm i && npm run lint
+
+  Test:
+    runs-on: ubuntu-latest
     strategy:
-      matrix:
-        node-version: [12.x]
+      fail-fast: false
 
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v1.4.4
       with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm install
-    - run: npm test
\ No newline at end of file
+        node-version: 14.x
+    - name: Install Dependencies
+      run: npm install
+    - name: Start Tests
+      run: npm run test
```

Comment:
Please do not edit configuration files for no reason

Reply:
I added tests.